packages/jaeger-ui/src/components/TracePage/TracePageHeader/TracePageHeader.tsx

```diff
@@ -149,15 +150,18 @@ export function TracePageHeaderFn(props: TracePageHeaderEmbedProps & { forwarded
       return { ...rest, value: renderer(trace) };
     });
 
+  const traceShortID = trace.traceID.slice(0, 7);
+
   const title = (
     <h1 className={`TracePageHeader--title ${canCollapse ? 'is-collapsible' : ''}`}>
-      <TraceName traceName={getTraceName(trace.spans)} />{' '}
-      <small className="u-tx-muted">{trace.traceID.slice(0, 7)}</small>
+      <TraceName traceName={trace.traceName} /> <small className="u-tx-muted">{traceShortID}</small>
     </h1>
   );
 
   return (
     <header className="TracePageHeader">
+      <Favicon url={trace.traceEmojiFaviconURL} />
```

Comment:
my understanding was that emoji will be the first char in the title, not replacing the whole Jaeger favicon. I don't like the latter - I think it breaks the implied contract between the website and the browser. 

Some websites alter the favicon (e.g. gmail shows actual msg count, others just add a red dot indicating unread messages).
<img width="36" alt="image" src="https://github.com/jaegertracing/jaeger-ui/assets/3523016/d532a8f2-173a-4082-a295-a4ba6fde78da">

 I wonder if this would work here, i.e. keep the overall favicon background as Jaeger logo but reserve a corner of it with a flat background and an emoji on top.

Reply:
The emoji is not legible if I do that. Could we make it a runtime setting whether we want to replace the favicon? My colleagues like the idea a lot and the favicon just wastes tab bar estate bringing no additional value for us. In any case, I made the favicon thing be its own commit at the very end of the list of commits, so we can easily apply the rest.

Reply:
yeah, we can add it to config. Meanwhile, I'd still suggest including emoji in the title (regardless of where it replaces favicon or not)

Reply:
Ack. I removed the last commit putting the emoji in a favicon. Will open another PR for configurable favicons.